Sources/Extraction/MinutiaeDetection/MinutiaeDetection.c

```diff
@@ -40,23 +40,24 @@ static MinutiaType GetMinutiaType(int numberOfNeighbors) {
   }
 }
 
-static List GetPointsWithNNeighbors(int n, BinaryMap * image, List minutiae)
+static void GetPointsWithNNeighbors(int n, BinaryMap * image, List* minutiae)
```

Comment:
Can we have the pointer * all in the same place? Either in the middle, or as a prefix to the variable (my preference).